test/converters/xcresult/xcresult.go

```diff
@@ -0,0 +1,188 @@
+package xcresult
```

Comment:
please create tests for this package i do not have a testSummary.plist, hard to understand and review the conversion without that

Reply:
there was tests for that already